kolibri/core/tasks/storage.py

```diff
@@ -524,6 +529,10 @@ def schedule(
         """
         if not isinstance(dt, datetime):
             raise ValueError("Time argument must be a datetime object.")
+        if repeat != None and repeat < 0:  # noqa E711
```

Comment:
In this case you should use:
```suggestion
        if repeat is not None and repeat < 0:
```
We're only forced to use `==` and `!=` for `None` when inside a SQLAlchemy query.

Reply:
Fixed sir.